lib/resources/r4/procedure.yaml

```diff
@@ -0,0 +1,51 @@
+---
+name: Procedure
+field_name_base_url: https://hl7.org/fhir/r4/procedure-definitions.html#Procedure
+fields:
+- name: status
+  required: 'Yes'
+  cardinality: 1..1
+  type: code
+  description: Codes identifying the lifecycle stage of an event.
+  url: https://hl7.org/fhir/r4/procedure-definitions.html#Procedure.status
+  binding:
+    description: Codes identifying the lifecycle stage of an event.
+    terminology:
+      - display: EventStatus
+        system: http://hl7.org/fhir/event-status
+        info_link: https://hl7.org/fhir/valueset-event-status.html
```

Comment:
This should be an R4 link

Reply:
Updated [here](https://github.com/cerner/fhir.cerner.com/pull/322/commits/5f721575a6963f19e39b8abaa2739ba52adf7a72).

Reply:
+1